lib/pipeline/addon/pipelines/project/pipelines/route.js

```diff
@@ -0,0 +1,11 @@
+import { inject as service } from '@ember/service';
+import Route from '@ember/routing/route';
+
+export default Route.extend({
```

Comment:
setDefaultRoute: on('activate', function() {
    this.set(`session.${C.SESSION.CONTAINER_ROUTE}`,'pipelines');
  }),```

and add route to app/authenticated/project/index/route.js VALID_ROUTES